test/error_test.py

```diff
@@ -34,10 +34,28 @@
 Traceback (most recent call last):
         ...
 samila.errors.samilaPlotError: Plotting process can't be Done because data2 is empty. Use generate method first.
+>>> g.generate()
+>>> g.plot(color=(1, 2, 3, 4, 5))
+Traceback (most recent call last):
+        ...
+samila.errors.samilaPlotError: Given color type is not supported.
 >>> g = GenerativeImage(lambda x,y: x, lambda x,y: y)
 >>> result = g.save_config()
 Traceback (most recent call last):
         ...
 samila.errors.samilaConfigError: Config file can't be saved. At least one of the function1_str or function2_str is None.
+>>> from samila.functions import *
+>>> select_color(2)
+Traceback (most recent call last):
+        ...
+samila.errors.samilaPlotError: Given color type is not supported. Replacing it with the default color.
```

Comment:
What?!
Replacing it with the default color?
It's an error!

Reply:
Yes, you're right.